pkg/cache/task.go

```diff
@@ -296,8 +303,7 @@ func (task *Task) postTaskAllocated(event *fsm.Event) {
 			zap.String("podUID", string(task.pod.UID)))
 
 		if err := task.context.apiProvider.GetAPIs().KubeClient.Bind(task.pod, nodeID); err != nil {
-			errorMessage = fmt.Sprintf("bind pod failed, name: %s, uid: %s, %#v",
-				task.pod.Name, task.pod.UID, err)
+			errorMessage = fmt.Sprintf("bind pod volumes failed, name: %s, %#v", task.alias, err)
```

Comment:
error does not implement GoStringer, why use `%#v`

Reply:
fixed